glotaran/parameter/parameter.py

```diff
@@ -409,6 +462,50 @@ def set_value_from_optimization(self, value: float):
         """
         self.value = np.exp(value) if self.non_negative else value
 
+    def markdown(
```

Comment:
This will be refactored in a later PR and properly tested. (Ref.: call with @joernweissenborn )